python/commons/virus.py

```diff
@@ -0,0 +1,183 @@
+import datajoint as dj
+
+schema = dj.schema('common_virus', locals())
+
+
+
+
+@schema
+class Gene(dj.Manual):
+    definition = """
+    # lookup table of virus genes
+
+    gene_name                   : char(30)      # identifier of the gne
+    ---
+    function=NULL               : varchar(1024) # anticipated function of that gene
+    dna_source=NULL             : varchar(255)  # source of the DNA
+    risk="no known risk"        : varchar(512) # risk for humans
+    """
+
+@schema
+class Construct(dj.Manual):
+    definition = """
+    # combinations of several genes
+
+    construct_id        : char(80)
+    """
+@schema
+class ConstructGene(dj.Manual):
+    definition = """
+    # genes in a construct
+
+    ->Construct
+    gene_name       : char(30) # name of the gene
+    """
+
+@schema
+class Type(dj.Lookup):
+    definition = """
+    # table of virus types
+
+    virus_type      : char(30)      # unique identifier for the virus type
+    ---
+
+    """
+
+    contents = [(t,) for t in ['AAV', 'Rabies', 'Lenti','Herpes']]
+
+
+@schema
+class Source(dj.Lookup):
+    definition = """
+    # table of vendors and sources of viruses
+
+    source_id        : varchar(20)  # source identifier
+    ---
+    """
+
+    @property
+    def contents(self):
+        yield from zip(['Penn', 'UNC', 'Homegrown', 'MIT'])
```

Comment:
I agree, but this is the legacy from the existing database. I'll add your suggestion and change once the other data is transferred.